index.js

```diff
@@ -8,23 +10,30 @@
 // }
 const FOOTNOTE_MAP = {}
 
-// @param {object} config - 11ty config
-// @param {object} [options] - Plugin options
-// @param {string} [baseClass] - Base CSS class for BEM
-// @param {string} [title] - Footnotes section title
-// @param {string} [titleId] - Footnotes section title ID
-// @param {func} [backLinkLabel] - Footnote back link label generator
+/** 
+ * @param {object} config - 11ty config
+ * @param {object} [options] - Plugin options
+ * @param {string} [baseClass] - Base CSS class for BEM
+ * @param {object} [classes] - custom class names for each element
+ * @param {string} [title] - Footnotes section title
+ * @param {string} [titleId] - Footnotes section title ID
+ * @param {func} [backLinkLabel] - Footnote back link label generator
+*/
 module.exports = (config, options = {}) => {
-  const baseClass = options.baseClass || 'Footnotes'
-  const title = options.title || 'Footnotes'
-  const titleId = options.titleId || 'footnotes-label'
-  const backLinkLabel =
-    options.backLinkLabel || ((_, index) => `Back to reference ${index + 1}`)
-  const cl = getClass(baseClass)
-
-  // @param {string} content - Footnote reference content
-  // @param {string} id - Footnote id
-  // @param {string} description - Actual footnote content
+  const { 
+    baseClass = 'Footnotes',
+    title = 'Footnotes',
+    titleId = 'footnotes-label',
+    backLinkLabel = ((_, index) => `Back to reference ${index + 1}`),
+    classes = {}
+  } = options;
```

Comment:
We can revert this; I just found it a little more compact since you can see the defaults at a glance

Reply:
Nah, it’s okay. I’m fine with that since it’s semantically equivalent. :)